packages/cardano-graphql/src/Schema/types/Transaction/AuxiliaryData/Script.ts

```diff
@@ -0,0 +1,19 @@
+// Review: seems like it doesn't include actual script in original cardano-graphql schema.
+// Should we add it?
```

Comment:
Yes we can for now. The main consideration for omitting this is storage size impact